addons/web/static/src/scss/form_view.scss

```diff
@@ -611,6 +611,10 @@
             padding-right: 25px;
         }
     }
+    .oe_form_field_html .o_field_translate {
```

Comment:
sorry, now it also impacts the inline editor that do not have the fullscreen button, so this was better before (but with space between plus)

![2020-07-02-130916_1444x557_scrot](https://user-images.githubusercontent.com/9977887/86351809-1fd51b00-bc65-11ea-8cba-6b18638dbb3f.png)